src/shared/components/loader/Loader.tsx

```diff
@@ -1,50 +1,20 @@
-import styled, { keyframes } from 'styled-components';
-import { variant as styledSystemVariant } from 'styled-system';
+import cn from 'classnames';
 
-type LoaderProps = React.ComponentPropsWithRef<typeof StyledLoader> & {
+type LoaderProps = React.ComponentPropsWithRef<'span'> & {
     variant: LoaderVariant;
 };
 
 type LoaderVariant = 'small' | 'big' | 'warning';
 
 export const Loader = ({ variant, ...rest }: LoaderProps) => {
-    return <StyledLoader variant={variant} {...rest} />;
+    return (
+        <span
+            className={cn('rounded-full animate-spin', {
+                'loader-small': variant === 'small',
+                'loader-big': variant === 'big',
+                'loader-warning': variant === 'warning',
```

Comment:
the classes could probably go inline here instead of needing to be put into `general.css` 🤔